sdk/eventhubs/azure_messaging_eventhubs/src/error.rs

```diff
@@ -1,27 +1,14 @@
 // Copyright (c) Microsoft Corporation. All Rights Reserved.
 // Licensed under the MIT License.
 
-use azure_core_amqp::{AmqpDescribedError, AmqpError};
+use azure_core_amqp::AmqpDescribedError;
 
 /// Represents the different kinds of errors that can occur in the Eventhubs module.
+#[derive(Debug)]
 pub enum ErrorKind {
-    /// An arithmetic overflow has occurred.
-    ArithmeticError,
-    /// An invalid parameter was passed to a function.
-    InvalidParameter(String),
-
-    /// The connection string is missing.
-    MissingConnectionString,
-
-    /// The shared access key name is missing.
-    MissingSharedAccessKeyName,
-
-    /// The endpoint is missing.
-    MissingEndpoint,
-
     /// The session was missing for the partition.
     MissingSession,
-
+    //
```

Comment:
???

Reply:
I commented out a bunch of stuff, then removed it. And missed one of the "commented out" lines.